### ISSUE TITLE
Can't create new passwords with 2.4.3 (Error when parsing key user id)

### ISSUE THREAD

**Issue by danpospisil** [passbolt/passbolt_browser_extension#52]
After updating the extension today to 2.4.3 no user is able to create a new password. We are stuck on 
```
Encrypting ...
Take a deep breath and enjoy being in the present moment...
```
We are using Passbolt API CE v 2.4.0 - we are using self-signed certificate with our own CA - it was working fine till yesterday. No other changes were made.

**Comment by cedricalfonsi** [comment]
Hello @danpospisil,

Can you please provide me with more information :
- Browser name and version
- Did you try to close the browser and restart it ? 

I cannot reproduce your issue on MacOs with Firefox (63.0.1) and chrome (70).

**Comment by cedricalfonsi** [comment]
Moreover can you copy/paste here any log you'll get in the browser console :
Firefox: https://developer.mozilla.org/en-US/docs/Tools/Browser_Console
Chrome: https://stackoverflow.com/a/10258029/4063145

**Comment by danpospisil** [comment]
@cedricalfonsi - it has stopped working for all of our users - despite of OS / Browser. I have tried it on MacOS, Windows and Linux - browsers Firefox 63.0.1, Chrome 70.0
Console is clean. I have ran it through Charles proxy and I see there is only request to gpgkeys.json?api-version=v1 - but I dont see it in developper tools in network section. And thats it - nothing more and I see the "Encrypting ..." screen.

**Comment by cedricalfonsi** [comment]
Ok. Are you available for a video call ? If yes send me a message at support@passbolt.com

**Comment by danpospisil** [comment]
@clns - yes, we identified it with @cedricalfonsi - its definitely the source of it. He is working on a fix.

**Comment by clns** [comment]
@danpospisil Ok good. I didn't know you identified it, cause the label still said "cannot reproduce".

**Comment by cedricalfonsi** [comment]
Fixed with v2.4.4
